public/js/helpers/chartsHelpers.js

```diff
@@ -21,8 +21,8 @@ const createYTotalsList = (partialsList) => {
 const convertToPercentage = (value, total) => total ? value * 100 / total : 0;
 
 // Converts the y value in a data pair to its percentage value (using the y totals value)
-const percentageDataSetPair = (pair, total) => {
-    return {'x': pair['x'], 'y': convertToPercentage(pair['y'], total)};
+const percentageDataSetPair = (pair, total) => { 
+    return { x: pair['x'], y: convertToPercentage(pair['y'], total), label: pair['label'] };
```

Comment:
minor: could use dot notation here, `pair.label` instead of `pair['label']` for example